resources/git-branch-stage-commit.md

```diff
@@ -0,0 +1,120 @@
+# Git Basics (branch, stage, commit)
+Suppose you have just forked and cloned a repository onto your local machine. You're interested in making some changes to it, then eventually requesting that your changes be merged back into that repository. How do you get there?
```

Comment:
Do we need to start with a brief summary of "what is git and what is it used for"?

Reply:
Also, do we need to say how to fork and clone? (since we're saying 'git basics', not sure where to start...)

Reply:
@codebyuma yeah I agree that we should have a brief summary of git (uses, overview) somewhere. 

Caroline is working on a separate md file with fork, clone, and PR. I wonder if we should put the git overview at the top of that since it comes first chronologically (at least here for our purposes)? and then just rename this file to branch, stage, commit (aka making changes). Thoughts on this?

Reply:
@srhboo sounds good to me!